backend/models/image.js

```diff
@@ -45,7 +46,7 @@ const Image = sequelize.define('image', {
 })
 
 Image.prototype.getEntry = function getEntry () {
-  return Entry.findOne({where: {entryType: IMAGE_ENTRY, entryId: this.id}})
+  return SinglePiece.findOne({where: {pieceType: IMAGE_ENTRY, pieceId: this.id}}).getEntry()
```

Comment:
Looking at this function now, it definitely doesn't work. Apparently its either not used or just not covered by the tests. I'll look into it.